Author: Denis Kristak (Inuits)
Removing 2to3 from setup.py as it in Python standard library.
diff -ruN suds-jurko-0.6_orig/setup.py suds-jurko-0.6/setup.py
--- suds-jurko-0.6_orig/setup.py	2014-01-21 20:06:02.000000000 +0000
+++ suds-jurko-0.6/setup.py	2023-10-11 18:27:22.025824389 +0100
@@ -91,8 +91,6 @@
     # distutils.setup() 'obsoletes' parameter not introduced until Python 2.5.
     extra_setup_params["obsoletes"] = ["suds"]
 
-if sys.version_info >= (3, 0):
-    extra_setup_params["use_2to3"] = True
 
     #   Teach Python's urllib lib2to3 fixer that the old urllib2.__version__
     # data member is now stored in the urllib.request module.
